src/common/options.cc

```diff
@@ -3951,6 +3951,10 @@ std::vector<Option> get_global_options() {
     .set_default(true)
     .set_description("Preextent rocksdb wal files on mkfs to avoid performance penalty"),
 
+    Option("bluefs_log_replay_check_allocations", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+      .set_default(true)
```

Comment:
maybe default this to false, actually?

Reply:
I'd prefer to have it on by default. Otherwise IMO it's going to be void. Are you expecting much overhead due to this?

Reply:
I guess not since it's only on replay.  I was thinking that we'd have it on in the default teuthology config.  This is fine, though!